mne_bids/read.py

```diff
@@ -182,10 +183,21 @@ def _read_events(events, event_id, raw, bids_path=None):
     return all_events, all_dur, all_desc
 
 
+def _safe_index(lst, val, *, allow_all=False):
+    try:
```

Comment:
Could you add just a tiny little docstring here? Just so a newcomer would immediately understand how this function makes indexing "safe"?

Reply:
Heh, that's probably a misnomer. It's more of an informative-fail than safe... I'll see if I can make a better name

Reply:
That was **excatly** my point ;)